examples/seismic/tti/operators.py

```diff
@@ -60,12 +72,22 @@ def Gzz_centered(field, costheta, sintheta, cosphi, sinphi, space_order):
 
 def Gzz_centered_2d(field, costheta, sintheta, space_order):
     """
-    2D rotated second order derivative in the direction z
-    :param field: symbolic data whose derivative we are computing
-    :param costheta: cosine of the tilt angle
-    :param sintheta:  sine of the tilt angle
-    :param space_order: discretization order
-    :return: rotated second order derivative wrt z
+    2D rotated second order derivative in the direction z.
+
+    Parameters
+    ----------
+    field : Function
+        Symbolic data whose derivative we are computing.
```

Comment:
"Input for which the derivative is computed"

Reply:
fixed